src/components/Buttons/Buttons.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { SortType } from '../../types/SortType';
+
+type Props = {
+  setSortby: (value: React.SetStateAction<SortType>) => void;
+  setIsReversed: (status: boolean) => void;
+  isReversed: boolean;
+  sortBy: SortType;
+};
+
+export const Buttons: React.FC<Props> = (
+  {
+    setSortby,
+    setIsReversed,
+    isReversed,
+    sortBy,
+  },
+) => {
+  const showResetBtn = () => {
+    return sortBy !== SortType.initial || isReversed;
+  };
+
+  return (
+    <div className="buttons">
+      <button
+        type="button"
+        className={`button is-info ${sortBy !== SortType.alphabet && 'is-light'}`}
```

Comment:
![image](https://github.com/mate-academy/react_list-of-goods-hooks/assets/70009205/b892ec22-1990-4ef5-8e28-9221c6cb4123)
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally
Fix all cases